test/integration/scenarios/network/peers_blacklist.js

```diff
@@ -65,7 +56,7 @@ module.exports = params => {
 							return done(err);
 						}
 
-						if ((numOfConnections - 20) <= expectedOutgoingConnections) {
+						if (numOfConnections - 20 <= expectedOutgoingConnections) {
```

Comment:
Looks like expectation, why we not using `expect` here?

Reply:
It is the way @ManuGowda and @nazarhussain have introduced in the other tests. It is the same behaviour.